README.md

```diff
@@ -210,21 +227,31 @@ article.submit()  # logged with "Article submitted" description
 
 There is an InlineForm available that can be used to display the history of changes.
 
-To use it expand your own `AdminModel` by adding `StateLogInline` to its inlines:
+To use it expand your own `AdminModel` by adding `PersistedTransitionInline` to its inlines:
 
 ```python
 from django.contrib import admin
-from django_fsm_log.admin import StateLogInline
+from django_fsm_log.admin import PersistedTransitionInline
 
 
 @admin.register(FSMModel)
 class FSMModelAdmin(admin.ModelAdmin):
-    inlines = [StateLogInline]
+    inlines = [PersistedTransitionInline]
+```
+
+### Migration to Abstract model PersistedTransitionMixin
+
+Once you defined your own model, you'll have to create the relevant migration to create the table.
+
+```sh
+python manage.py makemigrations
 ```
 
+Additionally you'd want to migrate the data from django_fsm_log.models.StateLog to your new table.
```

Comment:
Would be nice to have an example/snippet here that can be used as a base.

Reply:
Sure, it could be a nice addition, If I have to go through that path. I'll add it.
As of today I don't have incentive to migrate to an internal concrete model within the projects I maintain. It will come at some point, probably not as part of this PR though.